ase/search/models.py

```diff
@@ -3,6 +3,13 @@
 
 # Create your models here.
 class Search(models.Model):
+    """
+    -- Summary --
+    Search model has query details.
+    #Field Description
+    > query is a character filed with max possible length = 100, which is basically the user query.
```

Comment:
Try not to use general English and avoid specifying lengths/defaults
```suggestion
    > query is a character filed which keeps track of all the queries made onsite
```

Reply:
Done